extensions/amp-accordion/1.0/test-e2e/e2e-fixture.js

```diff
@@ -0,0 +1,6 @@
+import * as Preact from '#preact';
+import {render} from '#preact';
+
+import {_default as App} from '../storybook/Basic';
+
+render(<App />, document.querySelector('#bento'));
```

Comment:
This doesn't quite work when a Story has [`args`](https://storybook.js.org/docs/react/essentials/controls), since default values would be unset.

You'd have to do something like:

```suggestion
import config, {_default as App} from '../storybook/Basic';

render(<App {...getDefaultArgs(config, App)} />, document.querySelector('#bento'));
```

```js
function getDefaultArgs(config, fn) {
  const args = {...config.args, ...fn.args};
  const argTypes = {...config.argTypes, ...fn.argTypes};
  for (const [name, argType] of Object.entries(argTypes)) {
    args[name] = argType.defaultValue || args[name];
  }
  return args;
}
```